onwuvic/MyDiary
# #159115277 Setup and Refactor code with airbnb style guide

#### What does this PR do?
Ensure code quality using airbnb javascript style guide

#### Description of Task to be completed?
Having all code base clean and adhere to Airbnb style guide.

#### How should this be manually tested?
After cloning the repo and cd into it. RUN npm install to install dependencies
RUN npm run lint  -- to check if the code base adheres to Airbnb style guide.
RUN npm run lint:fix  -- to auto-fix, any of the codebase that didn't adhere to Airbnb style guide.
RUN npm run lint:watch  -- to allow eslint watch your codebase as you code.

#### What are the relevant pivotal tracker stories?
#159115277
